packages/neuron-ui/src/components/GeneralSetting/generalSetting.module.scss

```diff
@@ -33,6 +34,26 @@ $action-button-width: 11.25rem;
       }
     }
   }
+  .showVersion {
+    position: relative;
+
+    &::after {
+      content: attr(data-new-version-tip);
+      background-color: #ff1e1e;
+      border-top-right-radius: 8px;
+      border-bottom-left-radius: 8px;
+      position: absolute;
+      top: 0;
+      right: 0;
+      font-family: PingFang SC;
```

Comment:
This font is Apple-specific

Reply:
> This font is Apple-specific

<img width="1135" alt="image" src="https://github.com/nervosnetwork/neuron/assets/22511289/9818b636-f5a8-4f84-a525-ff78596000fc">

I'll ask our designer for help to handle it.